### PR TITLE
fixes game preference deep frying the game

### PR DESCRIPTION
**Pull request by dwasint**
## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

## Changelog
:cl:
fixes: game preferences no longer slowly deepfries players games
/:cl:

### CODE REVIEW

File: code/world.dm [hunk 1]
```diff
@@ -10,7 +10,7 @@
 	hub_password = "kMZy3U5jJHSiBQjr"
 	name = "BeeStation 13"
 	fps = 20
-	map_format = SIDE_MAP
```
Comment (itsmeow) [inline code/world.dm:13]: I thought we removed this already what

File: code/_onclick/hud/rendering/plane_master.dm [hunk 2]
```diff
@@ -31,16 +31,18 @@
...
 	appearance_flags = PLANE_MASTER
 	blend_mode = BLEND_MULTIPLY
 	alpha = 255
-/atom/movable/screen/plane_master/openspace
-	name = "open space plane master"
-	plane = OPENSPACE_PLANE
-	appearance_flags = PLANE_MASTER
+
+/atom/movable/screen/plane_master/openspace/backdrop(mob/mymob)
+	. = ..()
```
Comment (itsmeow) [inline code/_onclick/hud/rendering/plane_master.dm]: what are these changes for

Comment (dwasint) [reply code/_onclick/hud/rendering/plane_master.dm]: oops leftover change from downstream reverting that

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]
Fill out the PR body

**Pull request closed without merging**